lib/ssl/doc/src/using_ssl.xml

```diff
@@ -342,6 +342,38 @@ ssl:connect("localhost", 9999,
     
   </section>
 
+ <section>
+   <title>NSS keylog </title>
+   <p>The NSS keylog debug feature can be used by authorized users
+   to for instance enable wireshark to decrypt TLS packets.</p>
+
+   <p><em>Server (with NSS key logging)</em></p>
+    <code type="none">
+    server() ->
+        application:load(ssl),
+        {ok, _} = application:ensure_all_started(ssl),
+        Port = 11029,
+        LOpts = [{certs_keys, [#{certfile => "cert.pem", keyfile => "key.pem"}]},
+        {reuseaddr, true},
+        {versions, ['tlsv1.2','tlsv1.3']},
+        {keep_secrets, true} %% Enable NSS key log (debug option)
+        ],
+        {ok, LSock} = ssl:listen(Port, LOpts),
+        %% Accept first connection
+        {ok, CSock0} = ssl:transport_accept(LSock),
+        {ok, _} = ssl:handshake(CSock0),
+        %% Accept second connection
+        {ok, CSock1} = ssl:transport_accept(LSock),
+        {ok, Sock} = ssl:handshake(CSock1),
```

Comment:
explain somehow why 2 connections are needed?
keylog does not work for 1st connection?

maybe 2 connections are not needed and example can be simplified?

Reply:
Will look into if it can be simplified, I just split a old example into two. Thanks for the comment.

Reply:
Fixed